okd/scripts/provision.sh

```diff
@@ -2,10 +2,12 @@
 
 set -xe
 
-if [ ! -z $INSTALLER_RELEASE_IMAGE ]; then
-    until  export INSTALLER_COMMIT=$(oc adm release info $INSTALLER_RELEASE_IMAGE --commits | grep installer | awk '{print $3}' | head -n 1); do
-        sleep 1
-    done
+if [ -z $INSTALLER_COMMIT ]; then
```

Comment:
It can be useful when we want to use some specific commit, but if you only want to use `rc.4` release image you can just to specify the `$INSTALLER_RELEASE_IMAGE` and it will extract the installer commit from this image.

Reply:
yeah, the problem was the rc.4 installer didn't work. That's why I had to introduce a custom commit hash for the installer